content/en/profiler/enabling/ebpf.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Enabling the eBPF Profiler
+code_lang: ebpf
+type: multi-code-lang
+code_lang_weight: 90
+aliases:
+  - /tracing/profiler/enabling/ebpf/
+further_reading:
+    - link: 'getting_started/profiler'
+      tag: 'Documentation'
+      text: 'Getting Started with Profiler'
+    - link: 'profiler/profiler_troubleshooting/ebpf'
+      tag: 'Documentation'
+      text: 'Fix problems you encounter while using the profiler'
+---
+
+<div class="alert alert-warning">
+<code>ebpf</code> is in beta. Datadog recommends evaluating the profiler in a non-sensitive environment before deploying in production.
+</div>
+
+The eBPF profiler is our eBPF-based host profiler built on OpenTelemetry and designed to send profiling data to the Datadog backend via the Datadog Agent. It supports symbolication for compiled languages and is optimized for containerized environments like Docker and Kubernetes.
+
+## Requirements
+
+For a summary of the minimum and recommended runtime and tracer versions across all languages, read [Supported Language and Tracer Versions][7].
+
+Supported operating systems
+: Linux (glibc or musl)
+
+Supported architecture
+: `amd64` or `arm64` processors
+
+Serverless
+: `ebpf` is not supported on serverless platforms, such as AWS Lambda.
+
+OS Settings
+: `perf_event_paranoid` kernel setting is 2 or less (see [Troubleshooting][1])
+
+Debugging information
+: Symbols should be available. The profiler cannot provide human-readable function names if the symbol table is stripped.
```

Comment:
This is not the case since we support remote symbolication. Symbols should either:
* Be available locally (recommended)
* Be uploaded in CI via (datadog-ci) (we likely want a dedicated documentation page for this?)